.github/scripts/trymerge.py

```diff
@@ -1635,6 +1635,23 @@ def is_broken_trunk(
     )
 
 
+def is_unstable(
+    check: JobCheckState,
+    drci_classifications: Any,
+) -> bool:
+    if not check or not drci_classifications:
+        return False
+
+    name = check.name
+    job_id = check.job_id
+
+    # Consult the list of unstable failures from Dr.CI
+    return any(
+        (name == unstable["name"] or (job_id and job_id == unstable["id"]))
```

Comment:
are there any cases where you would actually expect the name != unstable['name'] but the id to be the same or vice versa?

Reply:
Nothing that I can think of.  Using the `job_id` is probably all we need, but I keep the name check for historical purpose as we have always done that before.  I think it could be cleaned up in a separate PR.